ProcessMaker/modeler
# FOUR-7959 Fix Selection with Data Store is not cloned

## Issue & Reproduction Steps
Selection with Data Store is not cloned. Steps to Reproduce: 

- Add Task form 
- Add Data Store
- Connect the Task Form with Data store 
- Select the elements 
- Press “Duplicate Selection“

### Expected behavior: 
The selection with data store should be cloned.

### Actual behavior: 
The selection with data store could not be cloned.

## Solution
The solution is to add a custom clone logic that handles properly the data store and dat object to be cloned.

## How to Test
To test the solution:
- add a Task form
- add a Data Store
- connect the Task Form with Data store
- select the elements
- and press “Duplicate Selection” to check if the selection with data store has been successfully cloned.

[CloneDataStoreDataObject.webm](https://user-images.githubusercontent.com/8028650/233383881-f1656bec-10c2-4538-893d-192405b0f2f7.webm)

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-7958
- https://processmaker.atlassian.net/browse/FOUR-7959

## Code Review Checklist
